packages/portal/src/app/shared/booking-form/booking-form.component.html

```diff
@@ -14,21 +14,21 @@
 
 <mat-divider></mat-divider>
 
-<section class="guests-container" [formGroup]="bookingForm" [ngClass]="{ 'filter-blur': isGuest() }">
+<section class="guests-container" [formGroup]="bookingForm"  [ngClass]="{ 'filter-blur': isGuest() }" >
   <mat-form-field appearance="outline">
     <mat-label>Enter renting period</mat-label>
-    <mat-date-range-input [formGroup]="rentingPeriod" [rangePicker]="picker" [min]="startFromToday()">
-      <input required matStartDate formControlName="start" placeholder="Start date" />
-      <input required matEndDate formControlName="end" placeholder="End date" />
+    <mat-date-range-input [formGroup]="rentingPeriod" [rangePicker]="picker" [min]="startFromToday()" [disabled]="isGuest()" >
+      <input required matStartDate formControlName="start" placeholder="Start date" [disabled]="isGuest()" />
+      <input required matEndDate formControlName="end" placeholder="End date" [disabled]="isGuest()" />
     </mat-date-range-input>
     <mat-hint>MM/DD/YYYY – MM/DD/YYYY</mat-hint>
     <mat-datepicker-toggle matSuffix [for]="picker"></mat-datepicker-toggle>
     <mat-date-range-picker #picker></mat-date-range-picker>
   </mat-form-field>
 
-  <mat-form-field appearance="outline">
+  <mat-form-field appearance="outline" >
```

Comment:
Great accessibility fix! Thank you very much! Please remove the additional spaces between the closing brackets and the attributes and we're good to merge!

Reply:
Sure !